src/sidebar/components/ShareDialog/ShareDialog.tsx

```diff
@@ -9,94 +10,94 @@ import ShareAnnotations from './ShareAnnotations';
 import TabHeader from './TabHeader';
 import TabPanel from './TabPanel';
 
+// The value for every item here represents the order in which tabs should be
+// rendered.
+// For example, if `share` has the lower value, it will be rendered as the first
+// tab when provided, even if it is provided last in the tabs prop.
+const tabsOrder = {
+  share: 0,
+  export: 1,
+  import: 2,
+} as const;
+
+export type ShareDialogTab = keyof typeof tabsOrder;
+
+export type ShareDialogProps = {
+  tabs: NonEmptyArray<ShareDialogTab>;
+};
+
+function capitalize<T extends string>(value: T): string {
+  const [firstLetter, ...rest] = value;
+  return `${firstLetter.toUpperCase()}${rest.join('')}`;
+}
```

Comment:
I would suggest to use a lookup (eg. via an object-as-map) to map the tab identifiers to strings. This will be easier to replace with translated lookups in future, when the identifier "share" maps to the French, Spanish etc. word for that.